hc-tree-view-behavior.html

```diff
@@ -0,0 +1,141 @@
+<link rel="import" href="../polymer/polymer.html">
+
+<script>
+  /* global hc */
+  // Ensure our namespace exists
+  window.hc = window.hc || {};
+
+  /**
+   * Use `hc.NestedListBehavior` to implement a flat data structure of nested items into a list with
+   *  children getting added/remove to emulate open/close functionality.
+   *
+   * @polymerBehavior hc.NestedListBehavior
+   */
+  hc.TreeViewBehavior = {
+    properties: {
+      itemIdName: {
+        type: String,
+        value: 'id'
+      },
+      itemParentIdName: {
+        type: String,
+        value: 'parentId'
+      },
+      /* The list of all selected items without duplicates */
+      selectedItems: {
+        type: Array,
+        value: () => []
+      },
+      /* The parts object used by the list element */
+      _parts: Object
+    },
+    listeners: {
+      'item-selected-changed': 'itemSelectedChanged',
+      'item-state-changed': 'itemStateChanged'
+    },
+    /**
+     * Takes an array of one or more items and removes them from the selectedItems array.
+     *
+     * @param {Array.<Object>} items
```

Comment:
This is an accepted format, and the recommended one when defining properties in Polymer for hydrolysis.

It uses the same list as type: http://usejsdoc.org/tags-type.html